.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -58,6 +58,15 @@ body:
     label: Last Known Working Electron version
     description: What is the last version of Electron this worked in, if applicable?
     placeholder: 16.0.0
+- type: dropdown
+  attributes:
+    label: Does the issue also appear in Chromium / Google Chrome?
+    description: If it does, please report the issue in the [Chromium issue tracker](https://issues.chromium.org/issues), not against Electron. Electron will inherit the fix once Chromium resolves the issue.
+    options:
+      - "Yes"
+      - "No"
```

```suggestion
      - Yes
      - No
```

Doesn't look like we use quotes in other parts of this file, so drop for consistency.

---

YAML 1.1 treats `yes` and `no` as booleans too, as well as `y` and `n`, as well as capitalised versions of them: https://yaml.org/type/bool.html

YAML 1.2 narrowed it to just `true` and `false` https://yaml.org/spec/1.2.2/#10212-boolean but many YAML parsers support both 1.1 and 1.2 for the sake of backwards compatibility.

So for them to be treated as strings instead of booleans they have to be quoted.

---

Ah, TIL. Thanks for the detailed explanation @absidue!

---

Yeah, I also had no idea until GitHub showed me that error. Thanks!